pkg/kubectl/stop.go

```diff
@@ -509,3 +507,7 @@ func (reaper *ServiceReaper) Stop(namespace, name string, timeout time.Duration,
 	}
 	return services.Delete(name)
 }
+
+func getNonOrphanDeleteOptions() *api.DeleteOptions {
+	return &api.DeleteOptions{OrphanDependents: new(bool)}
```

Comment:
i am guessing , this sets the OrphanDependents to false. Why not explicitly say false, rather having the reader guess that its false ? That will improve readability in imo